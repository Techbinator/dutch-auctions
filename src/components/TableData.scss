.table-data {
  span {
    width: 50%;
    display: inline-block;
    font-size: 1em;
    &:first-child {
      color: #999;
    }
    &:last-child {
      color: #000;
      font-weight: bold;
      text-align: right;
    }
    .error {
      width: 100% !important;
    }
  }
}
