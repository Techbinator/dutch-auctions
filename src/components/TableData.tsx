import React, { ReactNode } from "react";
import "./TableData.scss";
interface ITableData {
  title: string | ReactNode;
  value: string | ReactNode;
}
export default ({ title, value }: ITableData) => (
  <div className="table-data">
    <span>{title}</span>
    <span>{value}</span>
  </div>
);
