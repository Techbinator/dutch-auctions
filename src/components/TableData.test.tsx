import React from "react";
import TableData from "./TableData";
import { shallow } from "enzyme";

describe("TableData component", () => {
  it("should render snapshot with default values", () => {
    const tableData = shallow(
      <TableData title="Test title" value="Test value" />
    );
    expect(tableData).toMatchSnapshot();
  });
});
