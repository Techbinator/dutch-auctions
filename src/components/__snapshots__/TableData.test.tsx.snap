// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableData component should render snapshot with default values 1`] = `
<div
  className="table-data"
>
  <span>
    Test title
  </span>
  <span>
    Test value
  </span>
</div>
`;
